sdk/iothub/azure-resourcemanager-iothub/src/main/java/com/azure/resourcemanager/iothub/models/CertificateDescription.java

```diff
@@ -124,7 +124,7 @@
     CertificateDescription.Update update();
 
     /** The template for CertificateDescription update. */
-    interface Update extends UpdateStages.WithCertificate, UpdateStages.WithIfMatch {
+    interface Update extends UpdateStages.WithCertificate, UpdateStages.WithifMatch {
```

Comment:
I might need to double check codegen on this. Seems should be `WithIfMatch`.

Reply:
Fix integrated to pipeline.

However you might want wait for a few days for azure-identity May release done.